Makefile

```diff
@@ -0,0 +1,60 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the MIT License. See the LICENSE accompanying this file
+# for the specific language governing permissions and limitations under
+# the License.
+
+pkgname=amazon-ec2-net-utils
+version=1.5
+
+prefix=/usr/local
+sysconfdir=${DESTDIR}/etc
+sbindir=${DESTDIR}/${prefix}/sbin
+systemddir=${DESTDIR}/${prefix}/lib/systemd
+udevdir=${DESTDIR}/${prefix}/lib/udev
+mandir=${DESTDIR}/${prefix}/share/man
+
+# We support installation of some init-system dependent files, and
+# default to systemd.
+init=systemd
+
+define install-file
+test -d $2 || install -d -o root -g root -m755 $2
+install -o root -g root -m644 $1 $2
+endef
+
+define install-exe
+test -d $2 || install -d -o root -g root -m755 $2
+install -o root -g root -m755 $1 $2
+endef
+
+sources:
+	git archive --prefix ${pkgname}-${version}/ HEAD | gzip --best > ../${pkgname}-${version}.tar.gz
+
+test:
+	${MAKE} -C tests test
```

Comment:
Recursive Make Considered Harmful.

i.e. we can probably do the include thing instead and things should be okay

Reply:
OK. To me, recursive make here seemed quite a bit simpler, and the structure of the project as is doesn't really hit any of the classic issues with recursive make.